example/src/constants/Audios.ts

```diff
@@ -19,32 +19,47 @@ const filePath: string = globalMetrics.isAndroid
  * Copy a file to the specified destination path if it doesn't exist.
  * @param {string} value - The name of the file to copy.
  * @param {string} destinationPath - The destination path to copy the file to.
- * @returns {Promise<void>} A Promise that resolves when the file is copied.
+ * @returns {Promise<boolean>} A Promise that resolves to true if the file is copied successfully, otherwise false.
  */
 const copyFile = async (
   value: string,
   destinationPath: string
-): Promise<void> => {
+): Promise<boolean> => {
   const fileExists = await fs.exists(`${destinationPath}/${value}`);
 
   if (!fileExists) {
     try {
       const file = await fs.readFileRes(`raw/${value}`, 'base64');
       await fs.writeFile(`${destinationPath}/${value}`, file, 'base64');
+      return true;
     } catch (error) {
-      console.error(error);
+      console.error(`Error copying file ${value}: `, error);
+      return false;
     }
   }
+
+  return true; // File already exists
 };
 
 /**
  * Copy all files in the 'audioAssetArray' to the destination path (Android only).
- * @returns {Promise<void>} A Promise that resolves when all files are copied.
+ * @returns {Promise<string[]>} A Promise that resolves to a list of successfully copied file paths.
  */
-const copyFilesToAndroidResources = async (): Promise<void> => {
+const copyFilesToAndroidResources = async (): Promise<string[]> => {
```

Comment:
We can improve name of this function to something like `copyFilesToNativeResources` as we are also returning data for iOS as well.